pdc-client.spec

```diff
@@ -1,22 +1,35 @@
+# Enable Python 3 builds for Fedora
+# We could enable it for EPEL as well, but some dependencies (beanbag) are
+# missing there.
+# NOTE: do **NOT** change 'epel' to 'rhel' here, as this spec is also
+# used to do RHEL builds without EPEL
 %if 0%{?fedora}
-%global with_python3 1
+# If the definition isn't available for python3_pkgversion, define it
+%{?!python3_pkgversion:%global python3_pkgversion 3}
+%bcond_without  python3
 %else
+%bcond_with     python3
+%endif
+
+# Compatibility with RHEL. These macros have been added to EPEL but
+# not yet to RHEL proper.
+# https://bugzilla.redhat.com/show_bug.cgi?id=1307190
 %{!?__python2: %global __python2 /usr/bin/python2}
-%{!?python2_sitelib: %global python2_sitelib %(%{__python2} -c "from distutils.sysconfig import get_python_lib; print (get_python_lib())")}
+%{!?python2_sitelib: %global python2_sitelib %(%{__python2} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")}
+%{!?python2_sitearch: %global python2_sitearch %(%{__python2} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib(1))")}
 %{!?py2_build: %global py2_build %{expand: CFLAGS="%{optflags}" %{__python2} setup.py %{?py_setup_args} build --executable="%{__python2} -s"}}
 %{!?py2_install: %global py2_install %{expand: CFLAGS="%{optflags}" %{__python2} setup.py %{?py_setup_args} install -O1 --skip-build --root %{buildroot}}}
-%endif
 
 Name:           pdc-client
 Version:        1.2.0
-Release:        1%{?dist}
+Release:        4%{?dist}
```

Comment:
I remember Jan mentioned that our version should be 1.7.0-1, any consideration for this version?